src/docfx/template/MustacheTemplate.cs

```diff
@@ -199,15 +200,15 @@ private static bool IsTruthy(JValue value)
 
         private BlockToken? GetTemplate(string name)
         {
-            return _templates.GetOrAdd(name, key =>
+            return _templates.GetOrAdd(_baseDirectory.Concat(new PathString(name)), key =>
```

Comment:
>_baseDirectory.Concat(new PathString(name)) [](start = 39, length = 43)

just use the name as cache key to avoid useless computation

Reply:
maybe PathString would be better to keep constant with others

---
In reply to: [486780502](https://github.com/dotnet/docfx/pull/6531#discussion_r486780502) [](ancestors = 486780502)